src/pages/workspaces/workspace/Data.js

```diff
@@ -445,6 +459,8 @@ class WorkspaceDataContent extends Component {
       h(AutoSizer, { key: selectedDataType }, [
         ({ width, height }) => h(FlexTable, {
           width, height, rowCount: selectedData.length,
+          onScroll: y => saveScroll(0, y),
```

Comment:
I'm curious: why are we only saving Y position?

Reply:
Flextables only move vertically, so there isn't a reason to store the x position since it doesn't change here.

Reply:
Ah yes, good point.